src/build/bundled.js

```diff
@@ -2,23 +2,27 @@
 
   /* istanbul ignore next */
   if (typeof define === 'function' && define.amd) {
-    define(['backbone', 'underscore'], function(Backbone, _) {
-      return (root.Marionette = root.Mn = factory(root, Backbone, _));
+    define(['backbone', 'underscore', 'backbone-metal'], function(Backbone, _, Metal) {
+      return (root.Marionette = root.Mn = factory(root, Backbone, _, Metal));
     });
   } else if (typeof exports !== 'undefined') {
     var Backbone = require('backbone');
     var _ = require('underscore');
-    module.exports = factory(root, Backbone, _);
+    var Metal = require('backbone-metal');
```

Comment:
Actually I don't think we want to introduce Metal in the AMD or CJS here since it is bundled.  The dependencies here should just be backbone and underscore.